### PR TITLE
Do not encode files loaded from network as utf8

### PR DESCRIPTION
**Pull request by dyedgreen**
If binary (e.g. WASM) files are downloaded, the `fetch_string_once` method corrupts them by encoding them as utf-8.

I'm unsure how to add tests for this error specifically, so any help with that would be very welcome :sweat_smile:

see: #3528

(reproduce the error:)
```javascript
// fails without this change
import * as wasm from "https://deno.land/x/sqlite/build/sqlite.wasm";
console.log(wasm);
```

### CODE REVIEW

**Comment by dyedgreen** [comment]
Not sure why the format-test fails, it sees to pass locally :no_mouth:

**Comment by bartlomieju** [comment]
> Not sure why the format-test fails, it sees to pass locally 😶

Make sure to have latest stable Rust and rustfmt installed (1.41.0)

**Comment by lukahartwig** [comment]
> Not sure why the format-test fails, it sees to pass locally 😶

I had the same problem. Turns out I was on rust 1.41.0-nightly. It works with 1.41.0-stable though.

**Comment by dyedgreen** [comment]
Thanks @bartlomieju, @lukahartwig that seems to fix it :sweat_smile:

**Comment by bartlomieju** [comment]
@dyedgreen that looks good, but we really need a test case demonstrating this problem for wasm. Could you add an integration test to `cli/tests/integration_tests.rs` that ensures it?

**Comment by dyedgreen** [comment]
@bartlomieju I added a test that importa a WASM file via the network. Let me know if this looks good now 😅

File: cli/tests/055_import_wasm_via_network.ts.out [hunk 1]
```diff
@@ -0,0 +1 @@
+{ __data_end, __dso_handle, __global_base, __heap_base, __wasm_call_ctors, add, memory }
```
Comment (bartlomieju) [inline cli/tests/055_import_wasm_via_network.ts.out]: 👍

File: cli/http_util.rs [hunk 2]
```diff
@@ -300,7 +300,10 @@ mod tests {
     let client = create_http_client();
     let result = fetch_string_once(client, &url, None).await;
```
Comment (bartlomieju) [inline cli/http_util.rs]: Since `fetch_string_once` no longer fetches string but rather bytes could you change its name to `fetch_once`?

**Comment by dyedgreen** [comment]
Thanks for the review @bartlomieju! I made the requested changes; would you mind having another look? 😅

**Pull request merged**